src/vite-env-override.d.ts

```diff
@@ -43,12 +43,16 @@ declare module "*.webp" {
 declare module "*.svg" {
   import type * as React from "react";
 
-  export const ReactComponent: React.FunctionComponent<
-    React.SVGProps<SVGSVGElement> & { title?: string }
+  const ReactComponent: React.FunctionComponent<
+    React.ComponentProps<"svg"> & {
+      title?: string;
+      titleId?: string;
+      desc?: string;
+      descId?: string;
```

Comment:
what is this?

Reply:
I've added a comment. This was copied from `vite-plugin-svgr`'s [client types](https://github.com/pd4d10/vite-plugin-svgr/blob/v4.3.0/client.d.ts). The comment explains why this is needed.